tasks/taxon_id/task_kraken2.wdl

```diff
@@ -140,4 +145,84 @@ task kraken2_standalone {
       disks: "local-disk 100 SSD"
       preemptible: 0
   }
+}
+
+task kraken2_parse_classified {
+  input {
+    File kraken2_classified_report
+    File kraken2_report
+    String samplename
+    String? target_org
+    Int cpu = 4
+    Int disk_size = 100
+  }
+  command <<<
+
+    gunzip -c ~{kraken2_classified_report} > ~{samplename}.classifiedreads.txt
+
+    python3 <<CODE 
+    import pandas as pd 
+
+    # load files into dataframe for parsing
+    classifiedreads_table = pd.read_csv("~{samplename}.classifiedreads.txt", names=["Classified_Unclassified","Read_ID","Taxon_ID","Read_Len","Other"], header=None, delimiter="\t")
+    report_table = pd.read_csv("~{kraken2_report}", names=["Percent","Num_reads","Num_reads_with_taxon","Rank","Taxon_ID","Name"], header=None, delimiter="\t")
+
+    # create dataframe to store results
+    results_table = pd.DataFrame(columns=["Percent","Num_basepairs","Rank","Taxon_ID","Name"])
+    
+    # calculate total basepairs
+    total_basepairs = classifiedreads_table["Read_Len"].sum()
```

Comment:
should this be calculated from the full report table?

Reply:
it is! renamed the variable to be more accurate